icx/Cargo.toml

```diff
@@ -22,9 +22,9 @@ clap = "3.0.0-beta.1"
 delay = "0.3.1"
 hex = "0.4.2"
 humantime = "2.0.1"
-ic-agent = { path = "../ic-agent", version = "0.3" }
+ic-agent = { path = "../ic-agent", version = "0.4" }
```

Comment:
is there a reason `icx` didn't get a minor version upgrade? (I notice `icx-proxy` did)

Reply:
no reason will do.